R/strategise.R

```diff
@@ -0,0 +1,120 @@
+strategise <- function(options, db) {
+  cost <- function(baseline_settings, intervention_settings, intervention) {
+    population <- baseline_settings$population
+    procurement_buffer <- (intervention_settings$procureBuffer + 100) / 100
+    cost_per_N1 <- intervention_settings$priceNetStandard
+    cost_per_N2 <- intervention_settings$priceNetPBO
+    price_NET_delivery <- intervention_settings$priceDelivery
+    price_IRS_delivery <- intervention_settings$priceIRSPerPerson * population
+
+    costs_N0 <- 0
+    costs_N1 <- (price_NET_delivery + cost_per_N1) * (population / intervention_settings$procurePeoplePerNet * procurement_buffer)
+    costs_N2 <- (price_NET_delivery + cost_per_N2) * (population / intervention_settings$procurePeoplePerNet * procurement_buffer)
+    costs_S1 <- 3 * price_IRS_delivery
+    costs_N1_S1 <- costs_N1 + costs_S1
+    costs_N2_S1 <- costs_N2 + costs_S1
```

Comment:
I doubt performance here is going to be much of an issue, but since this method is just returning the cost for a single intervention,  I guess this is calculating more costs that it needs to. These could each be separate functions, and the switch statement could only invoke the one it needs?

Reply:
Redundancy removed as part of refactoring cost function.